.github/workflows/fmt.yml

```diff
@@ -0,0 +1,26 @@
+name: Static Analysis
```

Comment:
Let's name this file something like `lint.yml` or `linting.yml` or `static-analysis.yml`, since `clippy` is not a formatting linter.

Reply:
Switched to lint.yml / `Lint`